include/omp/parallel_execution_omp.h

```diff
@@ -338,11 +338,11 @@ class parallel_execution_omp {
       mpmc_queue<output_type> & output_queue) const;
 
   template <typename T, typename ... Others>
-  void do_pipeline(mpmc_queue<T> & in_q, mpmc_queue<T> & same_queue, Others &&... ops) const
+  void do_pipeline(mpmc_queue<T> &, mpmc_queue<T> &, Others &&...) const
```

Comment:
Do not remove parameter names from in-class declaration. Just move implementation outside.